Vizzuality/gfw
# Feat: map basemaps

## Overview

This PR includes a new panel that lets the user change basemaps. It's divided into 3 sections:

1. Boundaries selector – Changes the political boundaries of the map.
2. Labels selector - Changes the labels of the map.
3. Basemaps Grid - Changes the basemap, if/when selecting landsat the user also has the possibilty of changing the year he wants to see.

## Demo

slack channel
## Notes

Run `yarn` +  `yarn start`

## Testing

- Check that the user can enable/disable both the boundaries layers and the labels layers.
- Check that the boundaries layer doesn't appear in the legend.


- Check that the user can change the basemap.
- Check that the user can change the year of the landsat basemap.
- Check that after zoom 11 the landsat basemaps change from the pre-generated version to the generated-on-the-fly version.
